src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHost.cs

```diff
@@ -120,6 +120,10 @@ internal async Task RunAsyncCore(CancellationToken cancellationToken)
 
             _started = true;
 
+            // Users may want to configure the culture based on some ambient state such as local storage, url etc.
+            // If they have changed the culture since the initial load, fetch satellite assemblies for this selection.
```

Comment:
Is this comment still relevant? This *is* the initial load don'tn't?

Reply:
Loading satellite cultures happens twice. Once based on the culture .NET Wasm selects, which is downloaded before Program.MainAsync runs: https://github.com/dotnet/aspnetcore/blob/blazor-wasm/src/Components/WebAssembly/WebAssembly/src/Hosting/EntrypointInvoker.cs#L25-L29 and once after user code's executed. The initial load refers to the one in entry point invoker.

Do you think this could be better clarified?

Reply:
super yes :0